python/graphscope/client/session.py

```diff
@@ -1089,6 +1094,64 @@ def get_config(self):
         """Get configuration of the session."""
         return self._config_params
 
+    def backup_vineyard_depployment(self, graphIDs: str, path: str, pvc_name: str):
```

Comment:
The `graphIds` seems is a list, why there is a str? If you accept both str and list, please address this in the comment.
Also, prefer detailed comments for these two methods, if you would like people to understand and using these.